### PR TITLE
STP-3355 (1.3): Update URL for GSG

### PR DESCRIPTION
**Pull request by nrockershousen**
# Description

The link to the GSG for Papaya and future releases will all use the same vanity URL (https://www.hpe.com/support/ex-S-8000), which will always points to the most current version of the GSG in HTML format. This PR replaces the old vanity URL.

This change is only needed in `release/1.3` and `main`.

**NOTE:** This link will not work until the GSG is published to the support center for the Papaya release.

# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by rustydb — APPROVED** [review]

File: install/README.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ communicates with at once, or specific devices can be targeted for a firmware up
 
 > **IMPORTANT:**
 > Before FAS can be used to update firmware, refer to the 1.5 _HPE Cray EX System Software Getting Started Guide S-8000_
```
Comment (jld-dehn-hpe) [inline install/README.md:173]: Should "1.5" be included here? For Papaya, the guide is version 22.11 or edition 5. Is this a reference to an older version of the guide that should be removed?

```suggestion
> Before FAS can be used to update firmware, refer to the _HPE Cray EX System Software Getting Started Guide S-8000_
```

File: operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Non-compute node (NCN) personalization applies post-boot configuration to the
 HPE Cray EX management nodes. Several HPE Cray EX product environments outside
 of CSM require NCN personalization to function. Consult the manual for each
 product to configure them on NCNs by referring to the
-[1.5 HPE Cray EX System Software Getting Started Guide S-8000](https://www.hpe.com/support/ex-gsg)
+[1.5 HPE Cray EX System Software Getting Started Guide S-8000](https://www.hpe.com/support/ex-S-8000)
```
Comment (jld-dehn-hpe) [inline operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md:7]: Should "1.5" be included here? For Papaya, the guide is version 22.11 or edition 5. Is this a reference to an older version of the guide that should be removed?

```suggestion
[HPE Cray EX System Software Getting Started Guide S-8000](https://www.hpe.com/support/ex-S-8000)
```

File: operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md [hunk 3]
```diff
@@ -146,7 +146,7 @@ documents for additional information, because role documentation is updated more
 changes are introduced.
 
 Consult the manual for each product in order to change the default configuration by
-referring to the [1.5 HPE Cray EX System Software Getting Started Guide S-8000](https://www.hpe.com/support/ex-gsg)
+referring to the [1.5 HPE Cray EX System Software Getting Started Guide S-8000](https://www.hpe.com/support/ex-S-8000)
```
Comment (jld-dehn-hpe) [inline operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md:149]: Should "1.5" be included here? For Papaya, the guide is version 22.11 or edition 5. Is this a reference to an older version of the guide that should be removed?

```suggestion
referring to the [HPE Cray EX System Software Getting Started Guide S-8000](https://www.hpe.com/support/ex-S-8000)
```

File: operations/CSM_product_management/Perform_NCN_Personalization.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ to create a [configuration layer](../configuration_management/Configuration_Laye
 Products may supply multiple plays to run, in which case multiple configuration
 layers must be created. Consult the manual for each product to configure them on
 NCNs by referring to the [1.5 HPE Cray EX System Software Getting Started Guide
```
Comment (jld-dehn-hpe) [inline operations/CSM_product_management/Perform_NCN_Personalization.md:18]: Should "1.5" be included here? For Papaya, the guide is version 22.11 or edition 5. Is this a reference to an older version of the guide that should be removed?

```suggestion
NCNs by referring to the [HPE Cray EX System Software Getting Started Guide
```

File: operations/CSM_product_management/Perform_NCN_Personalization.md [hunk 5]
```diff
@@ -41,8 +41,8 @@ error. This error can be ignored.
 ### Add Layer(s) to the CFS Configuration
 
 CFS executes configuration layers in order. Refer to the _1.5 HPE Cray EX
```
Comment (jld-dehn-hpe) [inline operations/CSM_product_management/Perform_NCN_Personalization.md:43]: Should "1.5" be included here? For Papaya, the guide is version 22.11 or edition 5. Is this a reference to an older version of the guide that should be removed?

```suggestion
CFS executes configuration layers in order. Refer to the _HPE Cray EX
```

File: upgrade/README.md [hunk 6]
```diff
@@ -110,7 +110,7 @@ See [Security Hardening](../operations/CSM_product_management/Apply_Security_Har
 
 After completion of the validation of CSM health, the CSM product stream has been fully upgraded and
 configured. Refer to the `1.5 HPE Cray EX System Software Getting Started Guide S-8000`
```
Comment (jld-dehn-hpe) [inline upgrade/README.md:112]: Should "1.5" be included here? For Papaya, the guide is version 22.11 or edition 5. Is this a reference to an older version of the guide that should be removed?

```suggestion
configured. Refer to the `HPE Cray EX System Software Getting Started Guide S-8000`
```

**Review by jld-dehn-hpe — APPROVED** [review]

**Pull request merged**